File: FizzBuzz/FizzBuzz.c
#include <cs50.h>
#include <stdio.h>
#include <string.h>

int main(void)
{
    for (int i = 1; i <= 100; i++)
    {
        int c = 0;

        if (i % 3 == 0)
        {
            printf("Fizz");
            c++;
        }

        if (i % 5 == 0)
        {
            printf("Buzz");
            c++;
        }

        if (c == 0)
        {
            printf("%i", i);
        }

        printf("\n");

    }
}

File: OwnProj/fizzbuzz.py
x = int(input("X: "))
y = int(input("Y: "))
count = int(input("Count: "))

for i in range(1, count+1):
    p = ""
    if i % x == 0:
        p += "fizz"
    if i % y == 0:
        p += "buzz"
    if p == "":
        p += str(i)
    print(p)